quotes/templates/quotes/quotes_list.html

```diff
@@ -1,27 +1,60 @@
 {% extends 'internal/base.html' %}
+{% load static %}
+{% load i18n %}
+{% block header %}
+    <link rel="stylesheet" href="{% static 'quotes/css/quotes.css' %}">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Given a discussion @KSG-IT/design, we want to use material icons instead of font awesome. It should also be included in the base-html rather than here. 

But for now we'll just let this stand, and I'll create a PR adding material icons.